### DISCUSSION TITLE
Providing passwords via files is missing

### DISCUSSION THREAD

**Discussion by jsurkont** [keycloak/keycloak#10375] [Archive]
Environmental variables with the `_FILE` suffix (e.g. `KEYCLOAK_PASSWORD_FILE`, `DB_PASSWORD_FILE`) are missing in the new quarkus distribution. There is the new `KC_DB_PASSWORD` variable, however `KC_DB_PASSWORD_FILE` is not available. Is it intentional? Is there another way for reading passwords from files?

**Comment by ste1nstone** [comment]
Hi @jsurkont I just had the same issue and noticed there is another bug for this requierment #10918 
For this bug there is also a MR available

**Reply by jsurkont** [reply]
Thanks @ste1nstone Hopefully the MR is merged soon.
